apps/_pomodoro_app/pages/timer/[id]/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import { useRouter } from 'next/router'
+
+import io from 'socket.io-client'
+// TODO(toplenboren): remove hardcoded ENDPOINT value
+const ENDPOINT = "http://127.0.0.1:3001";
```

Comment:
better to take it from configs

Reply:
Check example here: https://github.com/8iq/nodejs-hackathon-boilerplate-starter-kit/blob/master/packages/%40core.next/apollo.jsx#L19-L25 (serverUrl)

Reply:
@tramakarov please check this out